src/prop/prop_engine.cpp

```diff
@@ -466,23 +448,19 @@ Result PropEngine::checkSat() {
 
   d_theoryProxy->postsolve(result);
 
-  if( result == SAT_VALUE_UNKNOWN ) {
+  if (result == SAT_VALUE_UNKNOWN)
+  {
     ResourceManager* rm = resourceManager();
     UnknownExplanation why = UnknownExplanation::INTERRUPTED;
-    if (rm->outOfTime())
-    {
-      why = UnknownExplanation::TIMEOUT;
-    }
-    if (rm->outOfResources())
-    {
-      why = UnknownExplanation::RESOURCEOUT;
-    }
+    if (rm->outOfTime()) why = UnknownExplanation::TIMEOUT;
+    if (rm->outOfResources()) why = UnknownExplanation::RESOURCEOUT;
```

Comment:
This change must be reverted to comply with our coding guidelines (no online if statements without braces).